roles/python/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+--- # This role ensures Python2.7 is installed and accessible for ansible
+- name: Install python 2.7
+  raw: apt-get update -qq && apt-get install -qq python2.7
```

Comment:
Why can't we use `apt` module?

Reply:
Ansible's modules use python 2.7, and at this point it isn't available.

Reply:
https://docs.ansible.com/ansible/2.5/modules/raw_module.html
> Executes a low-down and dirty SSH command, not going through the module subsystem. This is useful and should only be done in two cases. The first case is installing python-simplejson on older (Python 2.4 and before) hosts that need it as a dependency to run modules, since nearly all core modules require it.